Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -9,6 +9,7 @@ import MessageModalComponent from "@/components/modal/genericMessage.vue";
 import MedicationHistoryReportComponent from "@/components/report/medicationHistory.vue";
 import MSPVisitsReportComponent from "@/components/report/mspVisits.vue";
 import COVID19ReportComponent from "@/components/report/covid19.vue";
+import ImmunizationHistoryReportComponent from "@/components/report/immunizationHistory.vue";
```

Comment:
my changes to the reports.vue file may overlap with @FuriousLlama 's updates on "9730: Update Report Launch page" task.

Reply:
can we show these on the sample? 
![image](https://user-images.githubusercontent.com/33761012/102283440-491df680-3ee7-11eb-9e21-f013a8a3dc57.png)

Reply:
except exclude "Date Printed"

Reply:
completed